src/pe/mod.rs

```diff
@@ -190,9 +243,25 @@ impl<'a> PE<'a> {
                     certificate_table.virtual_address,
                     certificate_table.size,
                 )?;
+
+                #[cfg(feature = "pe_authenticode")]
+                {
+                    let start = certificate_table.virtual_address as usize;
+                    let end = start + certificate_table.size as usize;
+                    excluded_sections.certtable = Some(start..end);
+                }
+            }
+
+            #[cfg(feature = "pe_authenticode")]
+            {
+                excluded_authenticode_sections = Some(excluded_sections);
             }
         }
         Ok(PE {
+            #[cfg(feature = "pe_source")]
+            bytes,
+            #[cfg(feature = "pe_authenticode")]
```

Comment:
in general i do not like this pattern in rust code, where the struct size changes according to cfg flags; if we merge something like this would very much prefer/require no cfg flag here (and we already have too many cfg flags)